Dockerfile.oracle

```diff
@@ -76,6 +76,9 @@ RUN find config -name '*.sample' | xargs -I{} sh -c 'cp $1 ${1%.*}' -- {}
 RUN mkdir /tmp/config
 RUN cp -r /opt/homs/config/* /tmp/config
 
+RUN yarn install && bundle exec rails assets:precompile
+RUN rm -rf /opt/homs/node_modules/
```

Comment:
Do it in one `RUN` command, otherwise the image size will not be shrinked after `node_modules` removing.